mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java

```diff
@@ -285,7 +285,7 @@ public void onSuccess(List<Feed> feeds) {
 
         @Override
         public void onFailure(Exception e) {
-            Timber.d("Resources fetching failed.");
+            Timber.d("Resources fetching failed. 1");
```

Comment:
@garg-anuj It's not needed, it was for my local debugging. I forgot to remove it